auslib/web/admin/base.py

```diff
@@ -81,7 +81,8 @@ def add_security_headers(response):
     response.headers["X-Frame-Options"] = "DENY"
     response.headers["X-Content-Type-Options"] = "nosniff"
     response.headers["Strict-Transport-Security"] = app.config.get("STRICT_TRANSPORT_SECURITY", "max-age=31536000;")
-    response.headers["Access-Control-Allow-Headers"] = "Authorization"
+    response.headers["Access-Control-Allow-Headers"] = "Authorization, Content-Type"
+    response.headers["Access-Control-Allow-Methods"] = "OPTIONS, GET, POST, PUT, DELETE"
```

Comment:
Is it OK to set `Access-Control-Allow-Headers` and `Access-Control-Allow-Methods` but maybe not set `Access-Control-Allow-Origin` ?

Reply:
I believe it's valid, but any response that doesn't contain `Access-Control-Allow-Origin` will fail the CORS preflight check (which is what we want if it's coming from an invalid origin, of course).